at_server_status/lib/at_status_impl.dart

```diff
@@ -69,12 +69,14 @@ class AtStatusImpl implements AtServerStatus {
     // ignore: omit_local_variable_types
     AtStatus atStatus = AtStatus();
     atStatus.atSign = atSign;
-    await AtLookupImpl.findSecondary(atSign, _root_url, _root_port!)
+    await CacheableSecondaryAddressFinder(rootUrl!, rootPort!)
```

Comment:
only one instance of CacheableSecondaryAddressFinder has to be created. Otherwise we will end up hitting the root server everytime for the same atsign.

Reply:
Please review the latest changes